R/core_functions.R

```diff
@@ -44,53 +44,48 @@ get_life_stages <- function(transitions) {
 #'   over time (as indicated by NA value in 'j_day' column)
 #' @param table input predictors table
 #' @param pred string specifying the name of the predictor, e.g. "host_den"
+#'
```

Comment:
most of the changes in this file are just to interpret the updated `config` structure and ensure we get the same modeling results - so fairly low priority for review